sormas-backend/src/main/java/de/symeda/sormas/backend/labmessage/LabMessageFacadeEjb.java

```diff
@@ -179,6 +179,7 @@ public LabMessageDto toDto(LabMessage source) {
 			target.setTestReports(source.getTestReports().stream().map(t -> TestReportFacadeEjb.toDto(t)).collect(toList()));
 		}
 		target.setReportId(source.getReportId());
+		target.setSampleOverallTestResult(source.getSampleOverallTestResult());
```

Comment:
Please include this into the [LabMessageFacadeEjbMappingTest](https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/sormas-backend/src/test/java/de/symeda/sormas/backend/labmessage/LabMessageFacadeEjbMappingTest.java).